wlms/ircbridge.go

```diff
@@ -54,10 +54,41 @@ func (bridge *IRCBridge) Connect(messagesIn chan Message, messagesOut chan Messa
 			message: event.Message(),
 		}:
 		default:
-			log.Println("Message Queue full.")
+			log.Println("IRC Message Queue full.")
+		}
+	})
+	bridge.connection.AddCallback("JOIN", func(e *irc.Event) {
+		if e.Nick != bridge.nick {
```

Comment:
favor early return to avoid right drift: `if e.Nick == bridge.nick {\nreturn;\n}`. Same below.